.pre-commit-config.yaml

```diff
@@ -35,3 +35,7 @@ repos:
     hooks:
       - id: ruff
         args: ["--fix"]
+  - repo: https://github.com/pre-commit/mirrors-mypy
```

Comment:
I'm not sure about adding this to the pre-commit because pretty much every existing file is going to fail the check. Eventually, we'll get to the point where they pass, but that could be far in the future.

Reply:
Fair. Middle ground is that I added each file to the `exclude` list in pre-commit so it shouldn't fail anything. Then we can slowly opt in files as they level up but not get blocked in the meantime. I've tested this and it correctly ignores the files I've changed, but let me know if you run into an issue.

Reply:
That works. I [also see you can run](https://github.com/pre-commit/pre-commit.com/blob/main/sections/advanced.md#temporarily-disabling-hooks) `SKIP=mypy git commit -m "message"` to skip a specific pre-commit (unlike `--no-verify` which skips all pre-commits).